Source/WebKit/UIProcess/WebAuthentication/fido/U2fAuthenticator.cpp

```diff
@@ -176,7 +176,10 @@ void U2fAuthenticator::continueRegisterCommandAfterResponseReceived(ApduResponse
     switch (apduResponse.status()) {
     case ApduResponse::Status::SW_NO_ERROR: {
         auto& options = std::get<PublicKeyCredentialCreationOptions>(requestData().options);
-        ASSERT(options.rp.id);
+        if (!options.rp.id) {
+            U2F_RELEASE_LOG("continueRegisterCommandAfterResponseReceived: rp.id empty. Should not be.");
```

Comment:
```
    if (options.rpId.isEmpty())
        options.rpId = callerOrigin.domain();
```

This is guaranteed to exist because of this code in` AuthenticatorCoordinator.cpp,` but I don't see the harm in this logging.